### PR TITLE
Add FONLL recipe

### PR DESCRIPTION
**Pull request by brinick**

### CODE REVIEW

**Comment by ktf** [comment]
Can you please rebase on top of the current master and force push? I am checking the linter for the recipes

File: fonll.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package: FONLL
+version: "1.3.3"
+tag: "v1.3.3"
+source: https://github.com/brinick/fonll.git
+requires:
+  - lhapdf
+---
+#!/bin/bash -ex
+
+make ${JOBS+-j$JOBS} -f $SOURCEDIR/main/Makefile VPATH="$SOURCEDIR/main:\
```
Comment (ktf) [inline fonll.sh]: Please align the `\`

File: fonll.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                                                        $SOURCEDIR/hdmassive:\
+                                                        $SOURCEDIR/hdresummed:\
+                                                        $SOURCEDIR/phmassive:\
+                                                        $SOURCEDIR/phresummed:\
+                                                        $SOURCEDIR/common" fonlllha
+mkdir -p $INSTALLROOT/bin;
+cp *.o fonlllha $INSTALLROOT/bin;
```
Comment (ktf) [inline fonll.sh]: Why copying the `*.o` in `bin`? Maybe you want to use something like `$INSTALLROOT/share` or similar?

**Comment by dberzano** [comment]
OK I am testing it.

**Comment by dberzano** [comment]
@eniel

**Pull request merged**